index.html

```diff
@@ -26,7 +30,7 @@
         <div class="cell"></div>
         <div class="cell"></div>
       </div>
-      <div class="tile" data-row="r1", data-col="c1" data-val="2">2</div>
+      <!-- <div class="tile" data-row="r1", data-col="c1" data-val="2">2</div> -->
```

Comment:
It's generally considered a bad practice to leave around commented out code in a project. It's perfectly fine to disable some code while you're testing things out, but if you no longer need it, just remove it. You can always retrieve the line from your git history later.